dot/digest/digest.go

```diff
@@ -123,6 +123,8 @@ func (h *Handler) HandleDigests(header *types.Header) {
 	for i, d := range header.Digest.Types {
 		val, ok := d.Value().(types.ConsensusDigest)
 		if ok {
+			// TODO: handleConsensusDigest never gets called other than the first time. (In development as well)
+			// i.e., header.Digest does not have consensus digest. Why?
```

Comment:
remove - not every block has a consensus digest, usually only first block of an epoch